astropy/io/votable/exceptions.py

```diff
@@ -317,8 +317,7 @@ class W02(VOTableSpecWarning):
     VOTable 1.2 corrects this error in the specification.
 
     **References**: `1.1
-    <http://www.ivoa.net/documents/VOTable/20040811/REC-VOTable-1.1-20040811.html#sec:name>`__,
-    `XML Names <http://www.w3.org/TR/REC-xml/#NT-Name>`__
```

Comment:
@tomdonaldson , is it okay to remove this or do I need to find a replacement link here?